docs/run/run-jobs-batch.mdx

```diff
@@ -8,30 +8,36 @@ description: How to run quantum computing jobs in batch mode using Qiskit Runtim
 
 Batch mode can shorten processing time if all jobs can be provided at the outset. If you want to submit iterative jobs, use [sessions](sessions) instead.  Using batch mode has these benefits:
 
```

Comment:
Apparently we missed adding this to the batch section:

<img width="803" alt="image" src="https://github.com/Qiskit/documentation/assets/28284230/14a80937-1fc3-44bc-8ad8-e5a1fc394f03">

This applies to batch as well.